content/pt-br/docs/reference/glossary/taint.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Mancha
```

Comment:
I could not find a better translation for this term.
I'm wondering if we should follow the same approach from [here](https://github.com/kubernetes/website/pull/37655#discussion_r1013523176).